bcda/responseutils/writer.go

```diff
@@ -22,4 +23,99 @@ func WriteError(outcome *fhirmodels.OperationOutcome, w http.ResponseWriter, cod
 	}
 }
 
-func WriteCapabilityStatement() {}
+func CreateCapabilityStatement() *fhirmodels.CapabilityStatement {
+	usecoors := true
+	dt := time.Now()
+	baseurl := "<host>"
+	statement := &fhirmodels.CapabilityStatement{
+		Status:       "active",
+		Date:         &fhirmodels.FHIRDateTime{Time: dt, Precision: fhirmodels.Date},
+		Publisher:    "Centers for Medicare & Medicaid Services",
+		Kind:         "capability",
+		Instantiates: []string{"https://api.bluebutton.cms.gov/v1/fhir/metadata"},
+		Software: &fhirmodels.CapabilityStatementSoftwareComponent{
+			Name:        "Beneficiary Claims Data API",
+			Version:     "0.1",
+			ReleaseDate: &fhirmodels.FHIRDateTime{Time: dt, Precision: fhirmodels.Date},
+		},
+		Implementation: &fhirmodels.CapabilityStatementImplementationComponent{
+			Description: "",
+			Url:         baseurl,
+		},
+		FhirVersion:   "3.0.1",
+		AcceptUnknown: "extensions",
+		Format:        []string{"application/json", "application/fhir+json"},
+		Rest: []fhirmodels.CapabilityStatementRestComponent{
+			{
+				Mode: "server",
+				Security: &fhirmodels.CapabilityStatementRestSecurityComponent{
+					Cors: &usecoors,
+					Service: []fhirmodels.CodeableConcept{
+						{
+							Coding: []fhirmodels.Coding{
+								{Display: "OAuth", Code: "OAuth", System: "http://hl7.org/fhir/ValueSet/restful-security-service"},
+							},
+							Text: "OAuth",
+						},
+						{
+							Coding: []fhirmodels.Coding{
+								{Display: "SMART-on-FHIR", Code: "SMART-on-FHIR", System: "http://hl7.org/fhir/ValueSet/restful-security-service"},
+							},
+							Text: "SMART-on-FHIR",
+						},
+					},
+				},
+				Interaction: []fhirmodels.CapabilityStatementSystemInteractionComponent{
+					{
+						Code: "batch",
+					},
+					{
+						Code: "search-system",
+					},
+				},
+				Operation: []fhirmodels.CapabilityStatementRestOperationComponent{
```

Comment:
Maybe redundant, but should we also list this endpoint in the Capability Statement?  (`/metadata`)

Reply:
good point! I'll add this.